public/javascripts/devices-create.js

```diff
@@ -4,6 +4,22 @@ $(() => {
         return document.body.dataset.thingpediaUrl;
     }
 
+    function isIframe() {
+        try {
+            return window.self !== window.top;
+        } catch (e) {
+            return true;
+        }
+    }
+
+    function isCrossOrigin() {
+        try {
+            return window.frameElement !== null;
+        } catch (e) {
+            return true;
+        }
```

Comment:
Why do we need this try/catch? Can this code throw exceptions?

Reply:
In modern browsers it will not throw exceptions. In really old browsers yes.

Reply:
How old is old browser? I don't think we support really old stuff, because we use modern JS features without transpilation.

Reply:
Probably too much old (> 5y). Removed the try/catch, since without transpilation the JS features used will not be available.